app/constants/prisoner-relationships-enum.js

```diff
@@ -1,8 +1,36 @@
 module.exports = {
-  'husband-wife-civil': {'displayName': 'Husband, wife or civil partner'},
-  'partner': {'displayName': 'Partner'},
-  'parent-grandparent': {'displayName': 'Parent or grand-parent'},
-  'sibling': {'displayName': 'Brother or sister'},
-  'child': {'displayName': 'Son or daughter'},
-  'sole-visitor': {'displayName': 'Sole visitor'}
+  HUSBAND_WIFE_CIVIL: {
+    value: 'husband-wife-civil',
+    displayName: 'Husband, wife or civil partner'
+  },
+
+  PARTNER: {
+    value: 'partner',
+    displayName: 'Partner'
+  },
+
+  PARENT_GRANDPARENT: {
+    value: 'parent-grandparent',
+    displayName: 'Parent or grand-parent'
+  },
+
+  SIBLING: {
+    value: 'sibling',
+    displayName: 'Brother or sister'
+  },
+
+  CHILD: {
+    value: 'child',
+    displayName: 'Son or daughter'
+  },
+
+  SOLE_VISITOR: {
+    value: 'sole-visitor',
+    displayName: 'Sole visitor'
+  },
+
+  NONE: {
+    value: 'none',
+    displayName: ''
+  }
```

Comment:
scratch that, we can put a generic version rather than repeat same code in each enum